mitmproxy/proxy/layers/http/_http_h2.py

```diff
@@ -34,10 +34,12 @@ class BufferedH2Connection(h2.connection.H2Connection):
     """
 
     stream_buffers: collections.defaultdict[int, collections.deque[SendH2Data]]
+    stream_trailers: Dict[int, Any]
```

Comment:
While we're here, could we make this `List[Tuple[bytes, bytes]]` instead of `Any` please (and add type annotations below)? :)